### ISSUE TITLE
parsing list of dicts

### ISSUE THREAD

**Issue by m0ntassar** [gr0und-s3ct0r/niet#34]
facing errors when trying to get list of dicts
example yaml:

```
service: "something"
enable: true
notifications :
  - 
    type: "email"
    name: "dummy user"
    address: "dummy@domain.tld"
  - 
    type: "sms"
    name: "dummy user"
    address: "123456789"
  - 
    type: "messenger"
    name: "dummy user"
    address: "dummy@domain.tld"
```

**Comment by 4383** [comment]
Thanks for opening this issue.

We need to investigate.

**Comment by 4383** [comment]
Well, problem solved with #36 

test it:
```shell
niet notifications debug.yaml
{address: dummy@domain.tld, name: dummy user, type: email}
{address: '123456789', name: dummy user, type: sms}
{address: dummy@domain.tld, name: dummy user, type: messenger}
```
